docs/examples/viz_tesseract.py

```diff
@@ -0,0 +1,186 @@
+        projected_marix.append(projeced_mat3D)  # vertices to be proj (3D)
+
+    return np.array(projected_marix)
+
+###############################################################################
+# Now, We have 4D points projected to 3D. Let's define a function to connect
+# lines.
+
+
+def connect_points(verts3D):
+    lines = []
+    len_vert = len(verts3D)
+
+    for i in range(len_vert-1):
+        if i < 8:
+            lines.append(np.array([verts3D[i], verts3D[i+8]]))
+        if i == 7:
+            pass
+        else:
+            lines.append(np.array([verts3D[i], verts3D[i+1]]))
+        if i % 4 == 0:
+            lines.append(np.array([verts3D[i], verts3D[i+3]]))
+
+    for i in range(3):
+        lines.append(np.array([verts3D[i], verts3D[i+5]]))
+        lines.append(np.array([verts3D[i+8], verts3D[i+5+8]]))
+    return np.array(lines)
+
+
+###############################################################################
+# Creating a scene object and configuring the camera's position
+
+scene = window.Scene()
+scene.set_camera(position=(0, 10, -1), focal_point=(0.0, 0.0, 0.0),
+                 view_up=(0.0, 0.0, 0.0))
+showm = window.ShowManager(scene,
+                           size=(1920, 1080), order_transparent=True)
+showm.initialize()
+
+###############################################################################
+# Creating vertices and points actors
+
+verts3D = rotate4D(verts4D)
+if not wireframe:
+    points = actor.point(verts3D, colors=p_color)
+    point_verts = utils.vertices_from_actor(points)
+    no_vertices = len(point_verts)/16
+    initial_verts = point_verts.copy() - \
+        np.repeat(verts3D, no_vertices, axis=0)
+
+    scene.add(points)
+
+###############################################################################
+# Connecting points with lines actor
+
+lines = connect_points(verts3D)
+edges = actor.line(lines=lines, colors=e_color,
+                   lod=False, fake_tube=True, linewidth=4)
+lines_verts = utils.vertices_from_actor(edges)
+initial_lines = lines_verts.copy() - np.reshape(lines, (-1, 3))
+
+scene.add(edges)
+
+###############################################################################
+# Initializing text box to display the name
+
+tb = TextBlock2D(text="Tesseract", position=(900, 950),
+                 font_size=20)
+showm.scene.add(tb)
+
+###############################################################################
+# Define a timer_callback in which we'll update the vertices of point and lines
+# actor using `rotate4D`.
+
+
+def timer_callback(_obj, _event):
+    global verts3D, angle
+    verts3D = rotate4D(verts4D)
+    if not wireframe:
+        point_verts[:] = initial_verts + \
+            np.repeat(verts3D, no_vertices, axis=0)
+        utils.update_actor(points)
+
+    lines = connect_points(verts3D)
+    lines_verts[:] = initial_lines + \
+        np.reshape(lines, (-1, 3))
+    utils.update_actor(edges)
+
```

Comment:
you need to add a stopping criterium, the tutorial never stops so we can not generate the documentation